tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "assemblyscript/std/assembly.json",
+  "include": ["./**/*.ts"]
+}
```

Comment:
This is the magic `tsconfig.json` file. Not much needed.

Reply:
What that does is it links to the config that AS ships: 

https://github.com/AssemblyScript/assemblyscript/blob/master/std/assembly.json

You can see there it only incude the AS type defs.